### ISSUE TITLE
Editor Preview - Revert #40388 once core Iframe change is ready.

### ISSUE THREAD

**Issue by Addison-Stavlo** [Automattic/wp-calypso#43818]
Once https://github.com/WordPress/gutenberg/pull/21102 lands in core, the fix for the editor preview added in #40388 will no longer be necessary and can be reverted.

**Comment by sirreal** [comment]
Duplicate of #43886
